cmd/protecodeExecuteScan_test.go

```diff
@@ -404,8 +404,12 @@ func TestGetTarName(t *testing.T) {
 			"",
 			"abc.tar",
 		},
+		"with sha ": {
+			"ppiper/cf-cli@sha256:c25dbacb9ab6e912afe0fe926d8f9d949c60adfe55d16778bde5941e6c37be11",
+			"",
```

Comment:
is there a problem if somebody adds a version tag?

Reply:
I tested it and its not a problem if the user want to specify sha+version to the dockerImage being pulled